internal/versions/versions.go

```diff
@@ -119,14 +118,11 @@ func (h *handler) V2ListSupportedOpenshiftVersions(ctx context.Context, params o
 		if len(supportedArchs) > 1 {
 			if !checkedForMultiarchAuthorization {
 				checkedForMultiarchAuthorization = true
-				if err := h.ValidateAccessToMultiarch(ctx, h.authzHandler); err != nil {
-					if strings.Contains(err.Error(), "multiarch clusters are not available") {
-						continue
-					} else {
-						return common.GenerateErrorResponder(err)
-					}
+				var err error
+				hasMultiarchAuthorization, err = h.authzHandler.HasOrgBasedCapability(ctx, ocm.MultiarchCapabilityName)
+				if err != nil {
+					return common.GenerateErrorResponder(err)
```

Comment:
Do not return the original error to the user because it will expose the inner details of the security related call. I would replace the error with a message that states that you can not access multiarch cluster and has a 404 code

Reply:
Is there any reason to return an error here in this case? Or can we just skip this image if the ocm call fails (as we do for if the authorization comes back `false`?

If we're changing behavior we might as well think about everything while we're here :sweat_smile: 
@mkowalski ?

Reply:
I ask because no one was specifically asking for a multiarch cluster in this case, they were just asking what versions are available to them. So it seems a bit strange to error with something like "multiarch not allowed" if they don't actually care about multiarch.

Reply:
It’s correct here to say that we mustn’t return an error but rather transparently hide the image that user has no access to